src/java.base/share/classes/javax/net/ssl/ExtendedSSLSession.java

```diff
@@ -163,4 +164,99 @@ public List<SNIServerName> getRequestedServerNames() {
     public List<byte[]> getStatusResponses() {
         throw new UnsupportedOperationException();
     }
+
+    /**
+     * Generate Exported Keying Material (EKM) calculated according to the
+     * algorithms defined in RFCs 5705/8446.
+     * <P>
+     * Note RFC 5705 (for (D)TLSv1.2 and earlier) calculates different EKM
+     * values depending on whether {@code context} is null or non-null/empty.
+     * RFC 8446 (TLSv1.3) treats a null context as non-null/empty.
+     * <P>
+     * The {@code label} {@code String} will be converted to bytes using
+     * the {@link java.nio.charset.StandardCharsets#UTF_8}
+     * character encoding.
+     *
+     * @spec https://www.rfc-editor.org/info/rfc5705
+     *     RFC 5705: Keying Material Exporters for Transport Layer
+     *     Security (TLS)
+     * @spec https://www.rfc-editor.org/info/rfc8446
+     *     RFC 8446: The Transport Layer Security (TLS) Protocol Version 1.3
+     *
+     * @implSpec The default implementation throws
+     *           {@code UnsupportedOperationException}
```

Comment:
Missing period at end of sentence.

Reply:
Added.  I also readded:

     *           Classes derived from ExtendedSSLSession must implement
     *           this method.

which is also used in an earlier method (`getStatusResponses`) in this class.